meta-dts-distro/recipes-bsp/flashrom/flashrom_git.bb

```diff
@@ -6,6 +6,8 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=751419260aa954499f7abaabaa882bbe"
 
 DEPENDS = "pciutils libusb libftdi"
 
+PV = "dasharo-v1.2.2+git${SRCPV}"
```

Comment:
This line results in:

```
WARNING: preferred version git of flashrom not available (for item flashrom)
WARNING: versions of flashrom available: 1.2.1 dasharo-v1.2.2+git
WARNING: preferred version git of flashrom not available (for item flashrom)
WARNING: versions of flashrom available: 1.2.1 dasharo-v1.2.2+git
WARNING: preferred version git of flashrom not available (for item flashrom-dev)
WARNING: versions of flashrom available: 1.2.1 dasharo-v1.2.2+git
```

Reply:
https://github.com/Dasharo/meta-dts/pull/196/commits/5b6bd5a07f848b46904f0f5dfd50ad7046794d53